modules/monitoring-querying-metrics-for-user-defined-projects-as-a-developer.adoc

```diff
@@ -29,14 +29,14 @@ Developers can only use the *Developer* perspective and not the *Administrator*
 
 . Select the project that you want to view metrics for in the *Project:* list.
```

Comment:
In the preview, I see step 1. I think how it is written reads well. If you want all the steps to be consistent and start with the verb, `Select` you could rephrase step 1 to:

1. Select **Observe -> Metrics** from the **Developer** perspective in the OpenShift Container Platform web console. 

Note, I think users will not be thrown from this "inconsistency" so it's fine to leave as is. As a writer, the sentence structure difference in one step does kind of throw me, haha.

Reply:
Good point. I'll follow your suggestion for better consistency.